setup.cfg

```
@@ -21,3 +21,10 @@ exclude =
     */south_migrations/,
     */migrations/,
     */testsettings.py,
+
+
```

Bovenin mag het nosetests deel weg.

Kun je het isort deel hierdoor vervangen? Dan staat dat gelijk goed. Ik paste het maar even hier, de bestaande regels kan ik niet op reageren hier:

```suggestion
[isort]
atomic = true
force_alphabetical_sort = true
force_single_line = true
include_trailing_comma = true
line_length = 88
lines_after_imports = 2
multi_line_output = 3
use_parentheses = true
```